tests/performance-test/deploy/performance-test-tg.sh

```diff
@@ -1,21 +1,93 @@
 #!/bin/bash
 # Sets up the envirovnment for performance testing
+
```

Comment:
I'm not going to comment in this file since most of the same code appears up a level in performance-test.sh and we aren't focused on tg at the moment. I've given it a read-over and it seems sane, but a but out of place with the same logic in the other script. All comments on code in performance-test.sh apply to the same code here of course.

Reply:
A lot of the responsibility of this container has been shifted over to the performance-test.sh script, although this configuration does not necessarly show that yet. Once I get around to re-implementing tg, most of this text will disappear